cmd/main.go

```diff
@@ -141,6 +141,7 @@ var Options struct {
 	EnableElasticAPM            bool          `envconfig:"ENABLE_ELASTIC_APM" default:"false"`
 	WorkDir                     string        `envconfig:"WORK_DIR" default:"/data/"`
 	LivenessValidationTimeout   time.Duration `envconfig:"LIVENESS_VALIDATION_TIMEOUT" default:"5m"`
+	EnableDiskEncryption        bool          `envconfig:"ENABLE_DISK_ENCRYPTION" default:"true"`
```

Comment:
why do we need it both in main and inventory? seems it's used only in the inventory Config